tests_integration/test_data/projects/napp_templates_processors_v1/app/setup_script.sql

```diff
@@ -8,3 +8,4 @@
 
 CREATE OR ALTER VERSIONED SCHEMA <% ctx.env.schema_name %>;
 EXECUTE IMMEDIATE from '/another_script.sql';
+select 'ctx.native_app.name: <% ctx.native_app.name %>';
```

Comment:
this tests that even if the PDF is implicitly converted to v2, the template context is still v1

Reply:
but shouldn't we check the expansion somewhere then?

Reply:
We don't care about the expansion, we're just testing that arbitrary files in v1 projects can continue to use v1 template syntax. This test fails if the template context becomes converted implicitly (which is the original bug that Michel found)